e2e/cypress/integration/integrations/incoming_webhook/description_field_for_incoming_and_outgoing_webhooks_can_hold_500_characters_spec.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Should this file be removed?

Reply:
removed - sorry about that - issue when the name was altered.